Submodels/output_handling.py

```
@@ -0,0 +1,11 @@
+"""
+
+
+"""
+
+
+class output_handling:
+    def __init__(self, *args) -> None:
```

Ruff (0.8.2)

Missing type annotation for `*args`

(ANN002)